src/pyhf/modifiers/histosys.py

```diff
@@ -101,6 +101,8 @@ class histosys_combined:
     def __init__(
         self, modifiers, pdfconfig, builder_data, interpcode='code0', batch_size=None
     ):
+        default_backend = pyhf.default_backend
```

Comment:
Is this needed just to deal with the PyTorch warnings from https://github.com/scikit-hep/pyhf/issues/2554?

Reply:
it's needed in order to type `_histosys_mask` correctly as a tensor (see the below lines where it is being used). You could drop it, but then the typehints won't work due to the changes in typing in numpy.